connector-sdk/feel-wrapper/src/main/java/io/camunda/connector/feel/FeelConnectorFunctionProvider.java

```diff
@@ -55,32 +61,39 @@ public class FeelConnectorFunctionProvider extends JavaFunctionProvider {
           args ->
               new ValContext(
                   new Context.StaticContext(
-                      new Map.Map3<>(
+                      new Map.Map4<>(
+                          ERROR_TYPE_PROPERTY,
+                          BPMN_ERROR_TYPE_VALUE,
                           BPMN_ERROR_ARGUMENTS.get(0),
                           toString(args, 0),
                           BPMN_ERROR_ARGUMENTS.get(1),
                           toString(args, 1),
                           BPMN_ERROR_ARGUMENTS_WITH_VARS.get(2),
                           toContext(args, 2)),
                       Map$.MODULE$.empty())));
-  private static final java.util.Map<String, List<JavaFunction>> functions =
-      java.util.Map.of(
-          BPMN_ERROR_FUNCTION_NAME, List.of(BPMN_ERROR_FUNCTION, BPMN_ERROR_FUNCTION_WITH_VARS));
 
-  @Override
-  public Optional<JavaFunction> resolveFunction(String functionName) {
-    throw new IllegalStateException("Should not be invoked.");
-  }
-
-  @Override
-  public List<JavaFunction> resolveFunctions(String functionName) {
-    return functions.getOrDefault(functionName, Collections.emptyList());
-  }
+  // Incident
+  private static final String INCIDENT_FUNCTION_NAME = "incident";
+  private static final List<String> INCIDENT_FUNCTION_ARGUMENTS = List.of("message");
+  private static final JavaFunction INCIDENT_FUNCTION =
+      new JavaFunction(
+          INCIDENT_FUNCTION_ARGUMENTS,
+          args ->
```

Comment:
We wanted to go with `technicalError` or `jobError` as it will not immediately lead to an incident. Only after failing the retries. We also wanted to support variables as described in: https://github.com/camunda/connectors/issues/1383

Reply:
or just `failJob` wiith optional arguments.

Reply:
The function is now called `failJob`